test/test-project/src/test/editor/editorView-test.ts

```diff
@@ -36,6 +36,20 @@ describe('EditorView', () => {
         expect(editor.findWebElement).not.undefined;
     });
 
+    it('openEditor works with diff editor', async () => {
+        await view.openEditor('Untitled-2');
+
+        await new Workbench().executeCommand('File: Compare Active File With...');
+        const quickOpen = new QuickOpenBox();
+        await quickOpen.setText('Untitled-1');
+        await quickOpen.confirm();
+        await new Promise((res) => { setTimeout(res, 500); });
+        
+        const diffEditor = await view.openEditor('Untitled-2 ↔ Untitled-1') as DiffEditor;
+        await new Promise((res) => { setTimeout(res, 500); });
+        expect(diffEditor.getTitle()).equals('Untitled-2 ↔ Untitled-1');
```

Comment:
If the `view.openEditor` fails to identify a `DiffEditor` it will fall back to a `TextEdior`, that still has the same `getTitle` method. So the test would not fail in such case.

We need an `expect` more specific to `DiffEditor`. Something in line of `diffEditor.getOriginalEditor` not being undefined.

Reply:
Makes sense :+1: